CHANGES.txt

```diff
@@ -2,6 +2,67 @@
 Pyface Changelog
 ================
 
+Release 7.2.0
+=============
+
+Pyface 7.2.0 is a minor release which introduces a variety of bug fixes,
+documentation updates, and other code maintenance. The most significant change
+introduced by this release is the addition of undo/redo functionality from the
+undo subpackage copied from apptools.
+
+This release should still be compatible with Traits 6.0+. Users are encouraged
+to upgrade to Traits 6.1+ to stay current as future releases of Pyface will
+stop supporting Traits 6.0.
+
+Highlights of this release
+--------------------------
+
+* New :mod:`undo <pyface.undo>` subpackage that has been copied over from
+  apptools (soon to be deprecated in apptools)
+
+Detailed changes
+----------------
+
+Thanks to:
+
+* Aaron Ayres
+* Mark Dickinson
+* Kit Yan Choi
+* Rahul Poruri
+* Corran Webster
+
+Features
+
+* Copy Undo over from apptools (#813)
```

Comment:
From an ets perspective this falls under "Maintenance and code organization", but from just a `pyface` perspective it provides new functionality, so I placed it in "Features".  I'm happy to move it (or any of the other entries for that matter) if a reviewer thinks it is misplaced.